wagn/lib/wagn/commands.rb

```diff
@@ -36,19 +36,27 @@ def find_spec_file filename, base_dir
   end
 end
 
+LOAD_TASKS= {
+  'reseed' => 'reseed',
+  'seed' => 'create',
+  'load' => 'seed'
+}
```

Comment:
That sort of opens a bigger can of worms.  Is there a justification for tasks being defined in wagn/lib/wagn/tasks/X.rake vs. wagn/lib/tasks/X.rake ?  Can you make the case for any of them being there besides the obvious wagn.rake?

To change the namespace from wagn to decko, I would move wagn/lib/wagn/tasks/wagn.rake to wagn/lib/tasks/decko.rake along with the rest of the *.rake in the directory (same names).  Would that work for you?  I looked around in the gems directory too and there are really no examples of doing it like we currently have it.

I think it would be good to move this stuff to the new name.  I have no attachment to doing it now, but I think it goes well with what we'll have left  on this crazy branch.